pkg/uwb-dw1000/hal/uwb_dw1000.c

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_uwb_dw1000
+ * @{
+ *
+ * @file
+ * @brief       Glue code for running uwb-core for RIOT
+ * *
+ * @}
+ */
+
+#include "dpl/dpl.h"
+#include "dw1000/dw1000_phy.h"
+#include "dw1000/dw1000_hal.h"
+#include "uwb_dw1000.h"
+
+#include "uwb/uwb.h"
+#include "uwb/uwb_mac.h"
+
+#include "utlist.h"
+#include "thread.h"
+
+#ifndef LOG_LEVEL
+#define LOG_LEVEL   LOG_NONE
+#endif
+#include "log.h"
```

Comment:
I don't understand why you do that all over the place. The log level is something that is set globally using CFLAGS or by setting LOG_LEVEL make variable.
The only thing that should be done with `LOG_` functions is to include `log.h` but not to define a specific default value in each file.

Reply:
To enable logs locally.

Reply:
Ok, I understand. It seems to be used already in other places, so I won't insist.